.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  PyPI:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v2
+      - name: Install Python dependencies
+        run: python -m pip install --upgrade pip setuptools wheel twine
+      - name: Build dist packages
+        run: python setup.py sdist bdist_wheel
+      - name: Upload packages
+        run: python -m twine upload dist/*
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
```

Comment:
I propose to use https://github.com/pypa/gh-action-pypi-publish for that

Reply:
I usually don't use it, since it runs on a separate docker container and therefore is a little slower due to container startup. Besides that, it does literally the same thing, install twine upload the files, but in a separate container. So I really don't care 🤷 If you prefer the pypa action, I am happy to change it, however I haven't used it, so we'd need to test it once its merged.

Reply:
I usually don't use third party actions, but in this case it's the official action from pypa. But it doesn't matter. Just a suggestion.

Reply:
Let's leave it as is then. One dependency less than could break 😏

Reply:
@codingjoe https://github.com/django-hijack/django-hijack/runs/3231981371?check_suite_focus=true 
Wouldn't it be better to use pypa action?